.github/workflows/build-images.yaml

```
@@ -102,7 +108,7 @@ jobs:
     # nb: use format(..) to catch both inputs.skip = true AND inputs.skip = 'true'.
     if: ${{ format('{0}', inputs.skip) != 'true' }}
     needs: [ tags, vm-kernel ]
-    runs-on: [ self-hosted, large ]
+    runs-on: [ self-hosted, gen3, large ]
```

```suggestion
    runs-on: [ self-hosted, large ]
```

unintented change from rebasing, probably